u-boot/pkg.yaml

```diff
@@ -0,0 +1,26 @@
+name: u-boot
```

Comment:
Since u-boot has different configurations for different boards, we might want to name this `u-boot-libretech-all-h3-cc-h5`. Alternative is that we build all the variations under `u-boot`.

Reply:
interesting if we could have built multiple variants and could choose one or another

Reply:
Just to note:  there appear to be hundreds of variations.

Reply:
yeah :/ ... not sure what the best strategy is.

Reply:
For now, I think we can do is just build variants here, and add all of them to the installer image. We can figure out something better in the long run.